### DISCUSSION TITLE
Accounts - issue with Chrome browser<title>

### DISCUSSION THREAD

**Discussion by SilviaSaverino** [community/community#42295] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi guys I hope you are well.
I have an issue with Firefox and Chrome browser...
Long story short if i open my repository with Firefox, everything is showing fine. 
But if I were to open it with Chrome, is like my account is not even logged and I can't see a preview of my repositories etc...Here some pics. 

I don't know how to fix this - the only extension i have in Chrome is for gitpod, so nothing is interfering i guess - but hey, let me know your thoughts anyway please" thank you.

![Firefox](https://user-images.githubusercontent.com/104912124/208918162-b3e7fa5b-843f-4afe-9c31-740b91b0d70a.png)
![Chrome](https://user-images.githubusercontent.com/104912124/208918187-bcf57451-4fb0-41db-ae5e-8d62213fcd6b.png)

**Comment by sawyerloss** [comment]
Finding the ideal Valentine's card for your particular guy might be difficult. Many cards use the same same old phrases and lack imagination. Others are simply ridiculous. BOOMF https://boomf.com/collections/valentines-day-cards has the card you need to make this Valentine's Day one he'll never forget!
